test/test_examples_2d_unstructured_quad.jl

```diff
@@ -11,14 +11,22 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
 @testset "Unstructured Curve Mesh for Euler" begin
   @testset "elixir_euler_unstructured_quad.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_unstructured_quad.jl"),
-      l2   = [7.844127595829963e-5, 0.00011715233141491677, 0.00011715235371677694, 0.0002587793057642201],
-      linf = [0.007935754054894772, 0.011476103071539345, 0.01147611262431103, 0.025080478753547464])
+      l2   = [0.0002948925825934343, 0.0002517795581454954, 0.0002517795583556971, 0.0006357553368178441],
+      linf = [0.0049633337805587985, 0.005626613768364486, 0.005626613698727745, 0.01007957474197596],
+      polydeg = 3, tspan = (0.0, 0.5))
   end
 
   @testset "elixir_euler_unstructured_quad_periodic.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_unstructured_quad_periodic.jl"),
-      l2   = [0.00010720127811104554, 0.00012453133183340216, 0.00012453133183106435, 0.0002862523679720807],
-      linf = [0.005838648336111252, 0.008981476526964016, 0.008981476526997323, 0.019987154066631874])
+      l2   = [0.00010978828464875207, 0.00013010359527356914, 0.00013010359527326057, 0.0002987656724828824],
+      linf = [0.00638626102818618, 0.009804042508242183, 0.009804042508253286, 0.02183139311614468])
```

Comment:
These changes are expected, I guess? Are they related to changes in the mesh files?

Reply:
yes, because I changed the mesh files to match those that would get generated by the more current `HOHQMesh.jl` so the number of elements changed slightly for the gingerbread man and box around circle meshes. I also changed the order of the polynomial that represents the curves for the box with a twist. This changed the reference error values